content/content-strategy/manage-content-requests/publish-content/sections.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: nav/sections
+sections:
+  - Who to involve in publishing content
```

Comment:
Should be `Who to involve` to match the corresponding `nav/section`

Reply:
Trisha: done